src/main/kotlin/org/rust/ide/lineMarkers/RsGithubIssueLineMarkerProvider.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.lineMarkers
+
+import com.intellij.codeInsight.daemon.LineMarkerInfo
+import com.intellij.codeInsight.daemon.LineMarkerProvider
+import com.intellij.icons.AllIcons
+import com.intellij.ide.BrowserUtil
+import com.intellij.openapi.editor.markup.GutterIconRenderer
+import com.intellij.psi.PsiElement
+import org.rust.RsBundle
+import org.rust.ide.utils.import.isCore
+import org.rust.ide.utils.import.isStd
+import org.rust.lang.core.psi.RsMetaItem
+import org.rust.lang.core.psi.RsOuterAttr
+import org.rust.lang.core.psi.ext.*
+
+/**
+ * Provide an icon to open the GitHub issue page, e.g. `rust` related.
+ */
+class RsGithubIssueLineMarkerProvider : LineMarkerProvider {
+    override fun getLineMarkerInfo(element: PsiElement): LineMarkerInfo<*>? {
+        if (element !is RsOuterAttr || !element.containingCrate.isStd || !element.containingCrate.isCore) return null
```

Comment:
It doesn't work for API from `alloc` crate, for example, for `Vec` unstable methods. You can use `element.containingCrate.origin != PackageOrigin.STDLIB` approach to check if the crate is a part of stdlib